components/CampaignListItem/index.tsx

```diff
@@ -0,0 +1,121 @@
+import ArchiveOutlinedIcon from '@mui/icons-material/ArchiveOutlined'
+import { Box } from '@mui/material'
+
+const styledmainbox = {
+  display: 'flex',
+  flexDirection: 'column',
+  width: '1070px',
+  height: '100%',
+  padding: '10px',
+  backgroundColor: '#EAF9FF',
+}
+
+const styledmainbody = {
+  display: 'flex',
+  flexDirection: 'column',
+  justifyContent: 'space-between',
+  padding: '0px 20px 0px 20px',
+}
+
+const styledcontentbox = {
+  display: 'flex',
+  height: '300px',
+  backgroundColor: '#D9D9D9',
+}
+
+const styledbottombox = {
+  display: 'flex',
+  flex: 'auto',
+  flexDirection: 'row',
+  justifyContent: 'space-between',
+  alignItems: 'center',
+  width: '100%',
+  height: '67px',
+  backgroundColor: '#FFFFFF',
+  padding: '0px 20px 0px 20px',
+}
+
+const styledbottomitems = {
+  display: 'flex',
+}
+
+const styledinput = {
+  fontSize: '15',
+  fontWeight: '700',
+}
+
+type Props = {
+  CampaignName: String
+  CampaignDetails: String
+  Donors: Number
+  Amount: Number
+  Country: String
+  Published: Boolean
```

Comment:
can rename `published` to `campaignStatus` and make it an enum. so `campaignStatus` is now of type `CampaignStatus`. this enum can be placed in `constants/campaign.ts` which can hold all the constants related to campaign
```
enum CampaignStatus {
  DRAFT = 'Draft',
  PUBLISHED = 'Published',
  ARCHIVED = 'Archived',
}
```
you can refer to the screenshots below from the backend repo for the campaign schema
![photo_2022-12-11 01 18 15](https://user-images.githubusercontent.com/51109678/206867230-97b3047d-350a-4538-b4b0-f241f4b9a002.jpeg)

<img width="275" alt="Screen Shot 2022-12-11 at 1 18 22 AM" src="https://user-images.githubusercontent.com/51109678/206867223-89b2e2df-ad0a-4741-b233-ffb76c33b43c.png">